draft-ietf-quic-transport.md

```diff
@@ -4051,38 +4052,41 @@ Network routing and path elements can however change mid-connection; an endpoint
 MUST disable ECN if validation later fails.
 
 
-# Packet Size {#packet-size}
+# Datagram Size {#datagram-size}
 
-The QUIC packet size includes the QUIC header and protected payload, but not the
-UDP or IP headers.
+A UDP datagram can include one or more QUIC packets. The datagram size refers to
+the total UDP payload size of a single UDP datagram carrying QUIC packets. That
+is, the datagram size includes includes the QUIC headers and protected payload,
+but not the UDP or IP headers.
```

Comment:
Now that the name has changed, I don't think the extra explanation is necessary.

```suggestion
A UDP datagram can include one or more QUIC packets. The datagram size refers to
the total UDP payload size of a UDP datagram carrying one or more QUIC packets.
```

Reply:
I prefer the extra detail here, it's worth spelling it out precisely since we use the concept a fair bit below.  I'll take the "one or more QUIC packets" though.